### PR TITLE
Revert "Remove vestigial refresh token endpoints"

### PR DESCRIPTION
**Pull request by MatthewBemis**
Reverts broadinstitute/rawls#1119

### CODE REVIEW

File: core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala [hunk 1]
```diff
@@ -361,7 +363,7 @@ object Boot extends IOApp with LazyLogging {
             executionServiceServers.map(c => c.key -> c.dao).toMap,
             groupsToCheck = Seq(gcsDAO.adminGroupName, gcsDAO.curatorGroupName),
             topicsToCheck = Seq(gcsConfig.getString("notifications.topicName")),
-            bucketsToCheck = Seq(hammCromwellMetadata.bucketName.value)
+            bucketsToCheck = Seq(gcsDAO.tokenBucketName)
```
Comment (helgridly) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala:366]: both?

**Review by davidangb — APPROVED** [review]
revert button works!  
https://github.com/broadinstitute/rawls/compare/2a1a0e8fa9594725fe942da2722b25931b7acb30...revert-1119-mb-remove-tokens-bucket shows no diffs. :+1:

**Comment by MatthewBemis** [comment]
jenkins retest after google billing project outage resolved

**Comment by MatthewBemis** [comment]
jenkins retest - fiab start failure (rsync issue?) 

`11:14:19 rsync error: error in file IO (code 11) at receiver.c(393) [receiver=3.1.1]`

**Pull request merged**